book/website/history.md

```diff
@@ -0,0 +1,43 @@
+# History of The Turing Way project
+
+## Founding (2018 - 2019)
+
+*The Turing Way* is the flagship project of the Tools, Practices, and Systems programme.
```

Comment:
I'm not sure if this is the best sentence to start things with: 
1) that wasn't the case in the beginning and 
2) what does this tell anyone outside of the Turing?

Reply:
Yes - agree with @EstherPlomp - TPS came after The Turing Way!